tests/Integ/S3Context.php

```diff
@@ -6,19 +6,195 @@
 use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\Gherkin\Node\PyStringNode;
 use Behat\Gherkin\Node\TableNode;
+use Aws\S3\S3Client;
+use Aws\S3\PostObject;
+use Aws\S3\PostObjectV4;
 use GuzzleHttp\Client;
 use GuzzleHttp\Psr7;
 use PHPUnit_Framework_Assert as Assert;
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\StreamInterface;
 
 class S3Context implements Context, SnippetAcceptingContext
 {
     use IntegUtils;
 
+    const MB = 1048576;
+    const RESOURCE_POSTFIX = 'aws-test-integ-s3-context';
```

Comment:
This constant is only used once and is unlikely to be used in another method. You could just put this string directly in the `getResourceName` method.